src/helpers.tsx/helpers.tsx

```diff
@@ -0,0 +1,14 @@
+import usersFromServer from "../api/users";
+import todosFromServer from "../api/todos";
+
+export function getUser(userId: number) {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
```

Comment:
Ensure that components using the `getUser` function handle the case where it returns `null`. This is important to avoid potential runtime errors when accessing properties of a `null` user.